requirements.txt

```diff
@@ -4,11 +4,14 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
-git+https://github.com/Brown-University-Library/django-shibboleth-remoteuser.git#egg=django-shibboleth-remoteuser
-git+https://github.com/mitodl/mit-moira.git#egg=mit-moira
+--no-binary psycopg2
+
+-e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
+-e git+https://github.com/Brown-University-Library/django-shibboleth-remoteuser.git#egg=django-shibboleth-remoteuser
+-e git+https://github.com/mitodl/mit-moira.git#egg=mit-moira
```

Comment:
We need to manually remove the `-e` flags for the github dependencies to resolve python path issues when installing via salt.